pkg/registry/core/pod/etcd/eviction.go

```diff
@@ -116,6 +116,9 @@ func (r *EvictionREST) Create(ctx api.Context, obj runtime.Object) (runtime.Obje
 }
 
 func (r *EvictionREST) checkAndDecrement(namespace string, podName string, pdb policy.PodDisruptionBudget) (ok bool, err error) {
+	if pdb.Status.ObservedGeneration != pdb.Generation {
```

Comment:
Aah ok, thanks.
Why do you say that spec is immutable? Is there some validation rule that disallows updates?